pom.xml

```diff
@@ -309,19 +309,19 @@
         <version.org.apache.ws.security>2.2.3</version.org.apache.ws.security>
         <version.org.apache.ws.xmlschema>2.2.4</version.org.apache.ws.xmlschema>
         <version.org.apache.xalan>2.7.1.jbossorg-4</version.org.apache.xalan>
-        <version.org.bouncycastle>1.60</version.org.bouncycastle>
+        <version.org.bouncycastle>1.62</version.org.bouncycastle>
         <version.org.bytebuddy>1.9.5</version.org.bytebuddy>
         <version.org.codehaus.jackson>1.9.13</version.org.codehaus.jackson>
         <version.org.codehaus.jettison>1.3.8</version.org.codehaus.jettison>
-        <version.org.cryptacular>1.2.0</version.org.cryptacular>
+        <version.org.cryptacular>1.2.3</version.org.cryptacular>
         <version.org.eclipse.jdt.core.compiler>4.6.1</version.org.eclipse.jdt.core.compiler>
         <version.org.eclipse.microprofile.config.api>1.3</version.org.eclipse.microprofile.config.api>
         <version.org.eclipse.microprofile.health.api>1.0</version.org.eclipse.microprofile.health.api>
         <version.org.eclipse.microprofile.metrics.api>1.1</version.org.eclipse.microprofile.metrics.api>
         <version.org.eclipse.microprofile.opentracing>1.1</version.org.eclipse.microprofile.opentracing>
         <version.org.eclipse.microprofile.rest.client.api>1.0.1</version.org.eclipse.microprofile.rest.client.api>
-        <version.org.eclipse.yasson>1.0.3</version.org.eclipse.yasson>
-        <version.org.eclipselink.version>2.7.3</version.org.eclipselink.version>
+        <version.org.eclipse.yasson>1.0.4</version.org.eclipse.yasson>
+        <version.org.eclipselink.version>2.7.4</version.org.eclipselink.version>
```

Comment:
I don't think that the EclipseLink upgrade is needed but if it happens, that is fine.

Reply:
@scottmarlow If you want this upgrade please make it happen separately.